x-pack/platform/plugins/shared/onechat/public/application/components/tools/esql/visualize_esql/styles.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { css } from '@emotion/react';
+import type { UseEuiTheme } from '@elastic/eui';
+
+export const visualizationWrapper = (euiTheme: UseEuiTheme['euiTheme'], height: number) =>
+  css({
+    position: 'relative',
+    height,
+    overflow: 'visible',
+    '&:hover [data-test-subj="visualizationButtonActions"], &:focus-within [data-test-subj="visualizationButtonActions"]':
```

Comment:
Are we able to add class names to the specific buttons that you would like the hover to affect so we can be more specific without using the data-test-subj as a selector for hover?

You might be able to add something like this to the visualisations button container:
`className="visualization-button-actions"` and then in the styles in this file try:
`&:hover > * { ... }` this should affect the direct children of this wrapper so both the buttons - I haven't tested this myself tho, just a hypothesis 😄

Reply:
Fixed